### DISCUSSION TITLE
Customization of Color 1-8

### DISCUSSION THREAD

**Discussion by OtaStrom** [SubtitleEdit/subtitleedit#6019] [Ideas]
Hi. It would be great to allow adjusting the exact color for default colors 1-8.
Now it's forced to options Yellow, #FF0000, #00FF00, Cyan, Black, White, Orange, Pink.
For example, I prefer #FFF151 as yellow subtitles, so now I have a shortcut set up for color 1 - yellow, but then I have to Find and replace all "Yellow" with "#FFF151". It would be nice to be able to set it up directly.

**Comment by niksedk** [comment]
For now I think you can change the colors in Settings.xml

````xml
    <Color1>Yellow</Color1>
    <Color2>#FF0000</Color2>
    <Color3>#00FF00</Color3>
    <Color4>Cyan</Color4>
    <Color5>Black</Color5>
    <Color6>White</Color6>
    <Color7>Orange</Color7>
    <Color8>Pink</Color8>
````
